diff -urNp -x '*.orig' AcePerl-1.92.org/Makefile.PL AcePerl-1.92/Makefile.PL
--- AcePerl-1.92.org/Makefile.PL	2006-01-04 16:18:10.000000000 +0100
+++ AcePerl-1.92/Makefile.PL	2021-05-07 23:12:11.931623853 +0200
@@ -24,7 +24,7 @@ push @extlib,'Freesubs' if $choice >= 2;
 push @extlib,'RPC'      if $choice >= 3;
 
 print "\n";
-setup_sitedefs() if prompt("Do you want to install Ace::Browser? ","n") =~ /[yY]/;
+setup_sitedefs() if prompt("Do you want to install Ace::Browser? ","y") =~ /[yY]/;
 
 my $headers  = "./acelib/wh";
 WriteMakefile(
@@ -64,9 +64,9 @@ sub setup_sitedefs {
     $cgi_path  = $CGI_PATH;
     $html_path = $HTML_PATH;
   }
-  $conf_path ||= '/usr/local/apache/conf/ace';
-  $cgi_path  ||= '/usr/local/apache/cgi-bin/ace';
-  $html_path ||= '/usr/local/apache/htdocs/ace';
+  $conf_path ||= '/etc/httpd/ace';
+  $cgi_path  ||= '/home/services/httpd/cgi-bin/ace';
+  $html_path ||= '/home/services/httpd/html/ace';
 
   get_path("site-specific configuration files",\$conf_path);
   get_path("acebrowser CGI scripts",\$cgi_path);
@@ -147,7 +147,7 @@ sub get_path {
 
   $$pathref = expand_twiddles(prompt("Directory for the $description (~username ok):",$$pathref));
   return if -d $$pathref;
-  return if prompt("$$pathref does not exist.  Shall I create it for you?",'y') !~ /[yY]/;
+  return if prompt("$$pathref does not exist.  Shall I create it for you?",'n') !~ /[yY]/;
   mkpath($$pathref) or warn "Couldn't create $$pathref. Please create it before installing.\n";
 }
 
