--- Makefile.PL.orig	Mon Jul 15 02:43:55 2002
+++ Makefile.PL	Mon Jul 15 02:43:57 2002
@@ -23,7 +23,7 @@
 push @extlib,'RPC'      if $choice >= 3;
 
 print "\n";
-setup_sitedefs() if prompt("Do you want to install Ace::Browser? ","n") =~ /[yY]/;
+setup_sitedefs() if prompt("Do you want to install Ace::Browser? ","y") =~ /[yY]/;
 
 my $headers  = "./ace/wh";
 WriteMakefile(
@@ -60,9 +60,9 @@
     $cgi_path  = $CGI_PATH;
     $html_path = $HTML_PATH;
   }
-  $conf_path ||= '/usr/local/apache/conf/ace';
-  $cgi_path  ||= '/usr/local/apache/cgi-bin/ace';
-  $html_path ||= '/usr/local/apache/htdocs/ace';
+  $conf_path ||= '/etc/httpd/ace';
+  $cgi_path  ||= '/home/httpd/cgi-bin/ace';
+  $html_path ||= '/home/httpd/html/ace';
 
   get_path("site-specific configuration files",\$conf_path);
   get_path("acebrowser CGI scripts",\$cgi_path);
@@ -143,7 +143,7 @@
 
   $$pathref = expand_twiddles(prompt("Directory for the $description (~username ok):",$$pathref));
   return if -d $$pathref;
-  return if prompt("$$pathref does not exist.  Shall I create it for you?",'y') !~ /[yY]/;
+  return if prompt("$$pathref does not exist.  Shall I create it for you?",'n') !~ /[yY]/;
   mkpath($$pathref) or warn "Couldn't create $$pathref. Please create it before installing.\n";
 }
 
